src/main/java/duke/UI.java

```diff
@@ -0,0 +1,83 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class UI {
```

Comment:
Since UI is an acronym, I think it would be better for it to be named Ui to follow PascalCase